filebeat/tests/system/test_registrar.py

```diff
@@ -1293,7 +1440,8 @@ def test_ignore_older_state_clean_inactive(self):
 
         # Make sure state is written
         self.wait_until(
-            lambda: self.log_contains("Registry file updated. 0 states written."),
+            lambda: self.log_contains(
```

Comment:
Seems like you used a different pep setting then what we have in `make fmt` for the line length. It's not a big deal but in general would be nice to only have the changes related to the PR in.

Reply:
Noted. From next PR.

Reply:
Modified PEP8 config as per beats' requirements.